.github/workflows/vm-perf-comparison.yml

```diff
@@ -38,21 +38,28 @@ jobs:
 
       - name: init
         run: |
-          docker-compose -f docker-compose-runner.yml up -d zk
+          docker compose -f docker-compose-runner.yml up -d zk
```

Comment:
Out of curiosity: is this required for the needs of the PR?

Reply:
I suppose it is not, it is required in other places (there is no docker-compose within the zk container) so I've changed that here for consistency.

Reply:
it's definitely not "consistency", we have "docker-compose" in dozens of other places. if you need to call compose from compose that is itself running in docker, there's definitely something monumentally wrong going on and we need to fix this

Reply:
We have the zk container from which various operations using zk are performed.
When calling "zk test rust" I need to recreate a postgres container that the database-interacting unit tests will use. Ideologically the lifetime of this container should be limited to "zk test rust" execution, therefore it would be better to run a nested docker instead, but I suppose that is not supported.

> there's definitely something monumentally wrong going on

what is wrong with managing containers from other containers? AFAIK kubernetes does exactly that.